geotrek/common/templates/common/common_detail.html

```diff
@@ -1,4 +1,21 @@
 {% extends "mapentity/mapentity_detail.html" %}
+{% load i18n static mapentity_tags %}
+
+{% block relatedobjectspanel %}
+  {% block before_attachments_extra_tab_nav %}
+  <li class="nav-item">
+      <a class="nav-link" id="tab-related-objects" href="#related-objects" data-toggle="tab">
+          <i class="bi-grid"></i>
+          <span class="d-none d-sm-inline">{% trans "Related objects" %}</span>
+      </a>
+  {% endblock %}
+
+  {% block before_attachments_extra_tab_content %}
+  <div id="related-objects" class="tab-pane">
+      {% smart_include "detail" %}
+  </div>
+  {% endblock %}
+{% endblock relatedobjectspanel %}
```

Comment:
Move relatedobjectspanel block to MapEntity, to ensure we can create generic view in MapEntity to load its content